CONTRIBUTING.md

```diff
@@ -117,3 +117,18 @@ bazel test //src/python/deepgnn/graph_engine/snark/tests:python_test --test_outp
 ```
 
 This command will run all tests with prefix `test_sanity_neighbors_index`.
+
+# Create PR
+
+Follow these steps to create a pull request from a forked repo:
+1. Sync your repo with `https://github.com/microsoft/DeepGNN`.
```

Comment:
I think 1. should be how to create a fork of our repo, since it is their first time making a PR if they reference this.

Reply:
Thanks! Updated instructions and added reference to github docs.